### DISCUSSION TITLE
lithium dendrite growth simulation

### DISCUSSION THREAD

**Discussion by dangyi123** [idaholab/moose#22606] [Q&A Modules: Phase field] [answered]
I'm working on the phase-field simulation of lithium dendrite growth, the reference paper is below, 
![image](https://user-images.githubusercontent.com/90828631/200213930-b9853f32-d7de-4701-95ea-c37271a78918.png)
I feel a little bit confused about the code that using the ACInterface kernel to calculate that nonlinear AC interface term. (the second term is the right side)
![image](https://user-images.githubusercontent.com/90828631/200214142-c592754a-5b11-49fe-8149-647bee2d7c18.png)
The mobility and the kappa of this model is constant, according to the ACInterface.c and ACInterface.h, and the input code of ACInterface is below:
  [./AC_bulk]
    type = AllenCahn
    variable = eta
    f_name = FF
  [../]
I think the "variable_L" might be false,
![image](https://user-images.githubusercontent.com/90828631/200215140-dc47ea69-75bf-41f9-82d2-8853d2a2dced.png)

 DOES this term's strong form is just two constant number multiply a  second derivative of the order parameter just like the formula?
But HOW does the anisotropy manifest? 
I can't find the angle description just like the kobayaoshi snowflake case which is described in other papers.

**Comment by dangyi123** [comment]
@dschwen Sir!!!

**Reply by GiudGiud** [reply]
@amjokisaari

**Comment by vermaprk** [comment, marked as answer]
I think there is no anisotropy due to kappa but a Lagrange noise is inserted to break the symmetry.
ACInterface is mainly suited for planar growth

**Reply by biaogxb** [reply]
Do you know which kernel can reflect anisotropy from ‘kappa‘  ？

**Reply by vermaprk** [reply]
You can look into doi:10.1016/0167-2789(93)90120-P
The kernel are ACInterfaceKobayashi1/2
But it's for two phase system only
